Scripts/script-FindSimilarIncidents.yml

```diff
@@ -108,7 +108,7 @@ script: |-
   def get_incidents_by_keys(similar_incident_keys, incident_time, incident_id, hours_back, ignore_closed, max_number_of_results):
       similar_keys_query = " and ".join(map(lambda t: '%s:"%s"' % (t[0], t[1]), similar_incident_keys.items()))
       incident_time = parse_datetime(incident_time)
-      max_date = incident_time
+      max_date = incident_time + timedelta(hours=hours_back)
```

Comment:
all the changes in this file won't break backward?

Reply:
No.. it's just expand the query to search across more time (like the other scripts)